config/package_layout.json

```diff
@@ -3,7 +3,8 @@
     "analyzers": {
       "clangsa": "clang",
       "clang-tidy": "clang-tidy"
-    }
+    },
+    "clang-apply-replacements": "clang-apply-replacements"
```

Comment:
It would be good to extend the following documentation too: https://github.com/Ericsson/codechecker/blob/master/docs/package_layout.md